packages/web-domains/src/home/features/gather-member/components/GatherMemberProfile/GatherMemberProfile.tsx

```diff
@@ -27,27 +27,37 @@ export const GatherMemberProfile = ({ meetingId, member }: GatherMemberProfilePr
     >
       <Link href={`${meetingId}/about/${meetingMemberId}`}>
         <div css={{ alignItems: 'center', display: 'flex', flexDirection: 'column' }}>
-          <ProfileImage imageUrl={profileImageFileUrl} isConnection={isHandWaved} />
+          <ProfileImage imageUrl={profileImageFileUrl} isConnection={isHandWaved} isOnwer={isOwner} />
           <Txt typography="title2" css={{ marginTop: '12px' }}>
-            {name}
-            {isOwner && (
-              <Txt typography="title1" css={{ paddingLeft: '4px', bottom: '1px', position: 'relative' }}>
-                👑
-              </Txt>
-            )}
+            {isMe ? '나' : name}
           </Txt>
         </div>
       </Link>
     </li>
   );
 };
 
-const ProfileImage = ({ imageUrl, isConnection = false }: { imageUrl?: string; isConnection?: boolean }) => {
-  const borderStyles = isConnection ? { border: `3px solid ${colors.primary500}` } : {};
+const ProfileImage = ({
+  imageUrl,
+  isConnection = false,
+  isOnwer = false,
+}: {
+  imageUrl?: string;
+  isConnection?: boolean;
+  isOnwer?: boolean;
+}) => {
+  const borderColorStyles = isConnection ? { borderColor: `${colors.primary500}` } : {};
 
   return (
-    <span css={{ position: 'relative', borderRadius: '50%', ...borderStyles }}>
+    <span css={{ position: 'relative', borderRadius: '50%', border: '3px solid transparent', ...borderColorStyles }}>
```

Comment:
오 굳이 transparent를 추가한 이유가 있나용 ?

만약 정말 필요하다면 위 `borderColorStyle`은 제거하고
```suggestion
    <span css={{ position: 'relative', borderRadius: borderRadiusVariants.round, border: `3px solid ${isConnetion ? color.primary500 : 'transparent'}` }}>
```
도 할 수 있을 것 같고

isConnection이 되었을 때만 border를 하고 싶은거였다면 아래도 가능할 거 같아요

```suggestion
    <span  
        css={{
          position: 'relative',
          borderRadius: borderRadiusVariants.round,
          ...(isConnection && { border: `3px solid ${colors.primary500}` }),
      }}>
```

Reply:
왕관 아이콘이 새롭게 추가되었는데 border 유뮤에 따라서 높이가 달려저서 transparent로 적용을 했습니다!

Reply:
아하 그럼 위에 코드로 수정하시는 거 어떤가요 ?

Reply:
👞  반영하신 것 같이 봤는데 조아요 !